internal/lockfile/lockfile.go

```diff
@@ -28,21 +29,21 @@ func NewLockFile(clusterName string) *LockFile {
 }
 
 func (l *LockFile) Verify() error {
-	_, err := os.Stat(l.Path)
+	pid, err := os.ReadFile(l.Path)
 	if err == nil {
-		return ErrLockFileExists
+		return fmt.Errorf("%w %s", ErrLockFileExists, pid)
 	}
 
 	if !os.IsNotExist(err) {
-		return fmt.Errorf("error while checking lock file: %w", err)
+		return fmt.Errorf("error while checking lock file: \"%w\"", err)
 	}
 
 	return nil
 }
 
 func (l *LockFile) Create() error {
-	_, err := os.Create(l.Path)
-	if err != nil {
+	const perms = os.FileMode(0o666)
```

Comment:
:wave: @ralgozino there's a useful set of consts that you can use from https://github.com/sighupio/furyctl/blob/main/internal/x/io/fs.go#L24

Reply:
Ciao @Al-Pragliola!

Nice! I'll switch to that. 

Thank you! 🫂